### PR TITLE
Fix #44313 `bitnami/suitecrm:7` fails to come up due to group ownership set to `root`

### LINKED ISSUE #44313: SuiteCRM v7 Fails to work due to incorrect group ownership in `postunpack.sh`
### Name and Version

bitnami/suitecrm:7

### What architecture are you using?

None

### What steps will reproduce the bug?

1. in a linux debian 12 server
2. run the provided two line `docker compose` example commands, to get suitecrm 7 running
3. `mariadb` runs fine, however `suitecrm` exits after a few seconds.
4.  When you read the error logs, you see `access denied` error.
5. When you browse to the home page of the `suitecrm` web app, you get an `http` error code.

### What is the expected behavior?

You should be able to access the home page of the `suitecrm` web app.

### What do you see instead?

`Access denied`, and an `http` error code.

### Additional information

When I browse the suitecrm app directory inside the container, I notice most files are set to `daemon` owner, and `root` group.
**_When I set all files and directories to `daemon` owner and `daemon` group, this issue is resolved, `suitecrm` runs correctly!_**
I notice the owner and group for all files, is set inside `postunpack.sh`.
I'll make a PR to update `postunpack.sh`.

### PR DESCRIPTION
**Pull request by chris001**
### Description of the change
Fix the group ownership of all files and folders in the web app `bitnami/suitecrm:7`

### Benefits
Allows the web app to run.  Without my change, the app failed and exited.

### Possible drawbacks
None.

### Applicable issues

- fixes #44313

### CODE REVIEW

File: bitnami/suitecrm/7/debian-11/rootfs/opt/bitnami/scripts/suitecrm/postunpack.sh [hunk 1]
```diff
@@ -31,8 +31,8 @@ info "Configuring file permissions for SuiteCRM"
 ensure_user_exists "$WEB_SERVER_DAEMON_USER" --group "$WEB_SERVER_DAEMON_GROUP"
 for dir in "$SUITECRM_BASE_DIR" "$SUITECRM_VOLUME_DIR" "${SUITECRM_BASE_DIR}/tmp"; do
     ensure_dir_exists "$dir"
-    # Use daemon:root ownership for compatibility when running as a non-root user
-    configure_permissions_ownership "$dir" -d "775" -f "664" -u "$WEB_SERVER_DAEMON_USER" -g "root"
+    # Use daemon:daemon ownership for compatibility when running as a non-root user
+    configure_permissions_ownership "$dir" -d "775" -f "664" -u "$WEB_SERVER_DAEMON_USER" -g "$WEB_SERVER_DAEMON_USER"
```
Comment (jotamartos) [inline bitnami/suitecrm/7/debian-11/rootfs/opt/bitnami/scripts/suitecrm/postunpack.sh:35]: The WEB_SERVER_DAEMON_GROUP env var also exists

```suggestion
    configure_permissions_ownership "$dir" -d "775" -f "664" -u "$WEB_SERVER_DAEMON_USER" -g "$WEB_SERVER_DAEMON_GROUP"
```

**Review by jotamartos — CHANGES_REQUESTED** [review]
Thanks for your contribution. Could you please take a look at my comment?

File: bitnami/suitecrm/7/debian-11/rootfs/opt/bitnami/scripts/suitecrm/postunpack.sh (hunk 1 above)
Comment (chris001) [reply bitnami/suitecrm/7/debian-11/rootfs/opt/bitnami/scripts/suitecrm/postunpack.sh:35]: Yes I agree, this makes sense to use this `WEB_SERVER_DAEMON_GROUP` env var!

**Review by jotamartos — APPROVED** [review]

**Comment by jotamartos** [comment]
Version `7.13.4-debian-11-r31` includes your changes. Thanks for your contribution.

**Pull request merged**